operators/multiclusterobservability/pkg/config/config.go

```diff
@@ -217,11 +217,27 @@ const (
 	HubEndpointSaName          = "endpoint-observability-operator-sa"
 )
 
+const schemeHttps = "https"
+
 const (
 	OauthProxyImageStreamName      = "oauth-proxy"
 	OauthProxyImageStreamNamespace = "openshift"
 )
 
+const (
+	ClusterLogForwarderCRDName    = "clusterlogforwarders.logging.openshift.io"
```

Comment:
From my POV we can update these once https://github.com/rhobs/multicluster-observability-addon/pull/63 lands, let me know otherwise